src/app/collective-rewards/metrics/AllTimeRewardsMetrics.tsx

```diff
@@ -40,7 +40,7 @@ const TokenRewardsMetrics: FC<TokenRewardsMetricsProps> = ({
     0n,
   )
 
-  const totalRewardsInHuman = Number(formatBalanceToHuman(totalRewards))
+  const totalRewardsInHuman = Number(formatCurrency(totalRewards.toString(), symbol))
```

Comment:
`formatCurrency` returns a 2,0 which is not valid for Number and returns a NaN that fails since inside the `formatMetrics` you are calling again `formatCurrency` which is expecting a number.

Reply:
thanks. I thought I removed it everywhere. clearly not.

Reply:
done